File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--creating-a-template-by-using-content-controls_6.cs
                buildingBlockControl1 = this.Controls.AddBuildingBlockGalleryContentControl(
                    this.Paragraphs[1].Range, "buildingBlockControl1");
                buildingBlockControl1.BuildingBlockCategory = "Employee and Customer Information";
                buildingBlockControl1.BuildingBlockType = Word.WdBuildingBlockTypes.wdTypeCustom1;
                buildingBlockControl1.PlaceholderText = "Choose your first building block";

                buildingBlockControl2 = this.Controls.AddBuildingBlockGalleryContentControl(
                    this.Paragraphs[2].Range, "buildingBlockControl2");
                buildingBlockControl2.BuildingBlockCategory = "Employee and Customer Information";
                buildingBlockControl2.BuildingBlockType = Word.WdBuildingBlockTypes.wdTypeCustom1;
                buildingBlockControl2.PlaceholderText = "Choose your second building block";

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--creating-a-template-by-using-content-controls_6.vb
            BuildingBlockControl1 = Me.Controls.AddBuildingBlockGalleryContentControl( _
                Me.Paragraphs(1).Range, "buildingBlockControl1")
            BuildingBlockControl1.BuildingBlockCategory = "Employee and Customer Information"
            BuildingBlockControl1.BuildingBlockType = Word.WdBuildingBlockTypes.wdTypeCustom1
            BuildingBlockControl1.PlaceholderText = "Choose your first building block"

            BuildingBlockControl2 = Me.Controls.AddBuildingBlockGalleryContentControl( _
                Me.Paragraphs(2).Range, "buildingBlockControl2")
            BuildingBlockControl2.BuildingBlockCategory = "Employee and Customer Information"
            BuildingBlockControl2.BuildingBlockType = Word.WdBuildingBlockTypes.wdTypeCustom1
            BuildingBlockControl2.PlaceholderText = "Choose your second building block"